app/routes/events/my-sessions/index.js

```diff
@@ -0,0 +1,37 @@
+import Ember from 'ember';
+
+const { Route } = Ember;
+export default Route.extend({
+  titleToken() {
+    return this.l10n.t('Upcomming');
+  },
+  templateName: 'events/my-sessions/past',
+  model() {
+    return [
+      {
+        title    : 'Super cool JS',
+        state    : 'rejected',
+        event    : 'OS Tech',
+        status   : 'Rejected',
```

Comment:
This field will not be in the API response, but to avoid computing it in the template, we will add it as a computed property.